### ISSUE TITLE
Layer conv1_flowS has unknown engine

### ISSUE THREAD

**Issue by s0972456** [JingchunCheng/SegFlow#7]
Hi Jingchun,

I have been stucked with this runtime error for a while, I have installed the version included in the repository and the path should be set up correctly(although I am not 100% sure). But whenever I attempt to run the demo I get this error:

I1206 12:39:48.895972 22295 layer_factory.hpp:77] Creating layer conv1_flowS
F1206 12:39:48.895979 22295 layer_factory.cpp:69] Layer conv1_flowS has unknown engine.

Any advice will be greatly appreciated!

**Comment by wasidennis** [comment]
Hi,

Have you tried this: https://github.com/bqlabs/flownet/issues/1.

It looks like the problem is related to the CUDNN.

**Comment by s0972456** [comment]
Thank you so much! The problem is gone and now I can run the code smoothly!

And another irrelevant question... I tried the demo code and It doesn't appear to do any on-line training on the first image and mask, is there anyway I can enable online training using the first ground truth segment?

**Comment by wasidennis** [comment]
No problem and good to know it is running!

Currently, the code in this repository does not contain the online training part. If you want to finetune the model on the first frame, please take a look at our another repository for the example: https://github.com/JingchunCheng/Seg-with-SPN. They have different models but the methodology for finetuning is the same.

**Comment by s0972456** [comment]
Understood! Thank you so much for your quick response!
